src/renderer/src/prefComponents/theme/index.vue

```diff
@@ -54,17 +82,28 @@ import { usePreferencesStore } from '@/store/preferences'
 import { storeToRefs } from 'pinia'
 import { useI18n } from 'vue-i18n'
 import themeMd from './theme.md?raw'
-import { getAutoSwitchThemeOptions, themes as configThemes } from './config'
+import { themes as configThemes } from './config'
 import markdownToHtml from '@/util/markdownToHtml'
+import Bool from '../common/bool'
 import CurSelect from '../common/select'
 import Separator from '../common/separator'
+import Compound from '../common/compound'
 
 const themes = ref([])
 
 const { t } = useI18n()
 const preferenceStore = usePreferencesStore()
 
-const { autoSwitchTheme, theme, customCss } = storeToRefs(preferenceStore)
+const { followSystemTheme, lightModeTheme, darkModeTheme, theme, customCss } = storeToRefs(preferenceStore)
+
+const themeOptions = [
+  { label: 'Light', value: 'light' },
+  { label: 'Dark', value: 'dark' },
+  { label: 'Material Dark', value: 'material-dark' },
+  { label: 'One Dark', value: 'one-dark' },
+  { label: 'Ulysses', value: 'ulysses' },
+  { label: 'Graphite', value: 'graphite' }
+]
```

Comment:
Consider putting these in "configThemes" as part of `./config` instead to avoid options duplication